test/test_youtube_misc.py

```diff
@@ -21,6 +22,12 @@ def test_youtube_extract(self):
         assertExtractId('http://www.youtube.com/watch?v=BaW_jenozKcsharePLED17F32AD9753930', 'BaW_jenozKc')
         assertExtractId('BaW_jenozKc', 'BaW_jenozKc')
 
+    def test_youtube_geo_restricted(self):
+        dl = FakeYDL()
+        dl.params['extract_flat'] = True
+        ie = YoutubeIE(dl)
+        with self.assertRaises(GeoRestrictedError):
+            ie.extract('https://www.youtube.com/watch?v=qh_liphx2no')
```

Comment:
If anything, this should be an extractor test in `yt_dlp.extractor.youtube._video` with an `expected_exception`. (And if we were putting it here in the `test` module then it would need to be decorated with `@is_download_test` so that it's not run by default).

But even if we make this an extractor test, it's still not a very useful test because it may fail just because of where the test runner is located (e.g. in Belgium, where the video is not geo-restricted).

IMO just revert all the test code

Reply:
Reverted all the test code.